include/velocypack/Builder.h

```diff
@@ -546,22 +613,26 @@ class Builder {
   }
 
   template <typename T>
-  uint8_t* addInternal(std::string const& attrName, T const& sub) {
-    return addInternal<T>(attrName.data(), attrName.size(), sub);
+  uint8_t* addInternal(std::string const& attrName, uint64_t tag, T const& sub) {
+    return addInternal<T>(attrName.data(), attrName.size(), tag, sub);
   }
 
   template <typename T>
-  uint8_t* addInternal(StringRef const& attrName, T const& sub) {
-    return addInternal<T>(attrName.data(), attrName.size(), sub);
+  uint8_t* addInternal(StringRef const& attrName, uint64_t tag, T const& sub) {
+    return addInternal<T>(attrName.data(), attrName.size(), tag, sub);
   }
 
   template <typename T>
-  uint8_t* addInternal(char const* attrName, T const& sub) {
-    return addInternal<T>(attrName, strlen(attrName), sub);
+  uint8_t* addInternal(char const* attrName, uint64_t tag, T const& sub) {
+    return addInternal<T>(attrName, attrName == nullptr ? 0 : strlen(attrName), tag, sub);
   }
 
   template <typename T>
-  uint8_t* addInternal(char const* attrName, std::size_t attrLength, T const& sub) {
+  uint8_t* addInternal(char const* attrName, std::size_t attrLength, uint64_t tag, T const& sub) {
+    if(attrName == nullptr) {
```

Comment:
This check adds a branch to a hot function, so I would like to avoid it.
Is the PR going to add calls to `addInternal` with an `attrName` argument which is a nullptr? In this case, these calls should be fixed instead.

Reply:
No.